CMakeLists.txt

```diff
@@ -1,5 +1,48 @@
 # SPDX-License-Identifier: Apache-2.0
 
+option(BUILD_ON_OHOS OFF)
+
+if(BUILD_ON_OHOS)
+  #指定openharmony路径
+  set(OHOS_DIR_PATH "/home/openharmony/")
+
+  #指定交叉编译器路径
+  set(TOOLSCHAIN_PATH "${OHOS_DIR_PATH}prebuilts/clang/ohos/linux-x86_64/llvm/")
```

Comment:
Since this should be specified by users. Maybe we should move the OHOS part to another CMakeLists.txt and then include our original one.